### PR TITLE
Simplify build, add xenial, Fix #16

### LINKED ISSUE #16: #15 breaks itest (incorrect usage of safe_load)
```
All scripts are in the path...
Failed to parse YAML from /nail/etc/services/fake_runs_on_1/service.yaml
Traceback (most recent call last):
  File "/usr/bin/services_deployed_here", line 20, in <module>
    for service in sorted(service_configuration_lib.services_deployed_here()):
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 218, in services_deployed_here
    return set(list_extra_deployed_here() + services_deployed_on(hostname))
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 222, in services_deployed_on
    service_configuration = read_services_configuration()
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 151, in read_services_configuration
    service_info = read_service_configuration_from_dir(rootdir, service_dirname)
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 127, in read_service_configuration_from_dir
    service_information = read_service_information(service_file)
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 82, in read_service_information
    return _read_yaml_file(service_file)
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 90, in _read_yaml_file
    data = load_yaml(fd.read())
  File "/usr/lib/python2.7/dist-packages/service_configuration_lib/__init__.py", line 62, in load_yaml
    return yaml.safe_load(fd, Loader=yaml.CLoader)
TypeError: safe_load() got an unexpected keyword argument 'Loader'
Service fake_runs_on_1 ISN'T showing up in services_deployed_here but should be
make: *** [itest_xenial] Error 1
```

### PR DESCRIPTION
**Pull request by asottile**

### CODE REVIEW

File: dockerfiles/lucid/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,3 @@
...
-
-RUN apt-get clean all
-RUN apt-get update
-
-RUN apt-get -y install dpkg-dev python-tox python-setuptools python-yaml
-RUN apt-get -y install python-dev debhelper
-RUN apt-get -y install ruby rubygems ruby-dev libopenssl-ruby rubygem-fpm
-RUN gem install fpm
-
-RUN ln -s /usr/bin/python /usr/local/bin/python
```
Comment (asottile) [inline dockerfiles/lucid/Dockerfile]: idk why we have this in a lot of our dockerfiles, it's entirely wrong :(

File: package-python [hunk 2]
```diff
@@ -24,11 +24,6 @@
 ## Package a twisted and add some extra fpm options
 ## ./package-python yelp1 --replace python-twisted-core python-twisted
 
-# The shebangs inside the source python scripts should contain '/usr/bin/env python' which will then
-# get resolved by setup.py. We want those to resolve to /usr/bin/python and not /usr/local/bin/python or
-# any other python we might have kicking around in our path
-export PATH=/usr/sbin:/usr/bin:/sbin:/bin
-
```
Comment (asottile) [inline package-python]: this was only here to work around the mistake in the dockerfile above

File: service_configuration_lib/__init__.py [hunk 3]
```diff
@@ -23,6 +23,10 @@
 
 import yaml
 
+try:
+    from yaml.cyaml import CSafeLoader as Loader
+except ImportError:  # pragma: no cover (no libyaml-dev / pypy)
+    Loader = yaml.SafeLoader
```
Comment (asottile) [inline service_configuration_lib/__init__.py]: this fixes #16

File: tests/ubuntu.sh [hunk 4]
```diff
@@ -40,7 +40,10 @@ SERVICES_NOTHING="fake_total_bunk"
 mkdir -p /nail/etc
 [ -L /nail/etc/services ] || ln -s /work/tests/fake_services /nail/etc/services
 
-if dpkg -i /work/dist/*.deb; then
+apt-get update
+DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends gdebi-core
+
+if gdebi -n /work/dist/*.deb; then
```
Comment (asottile) [inline tests/ubuntu.sh]: yay resolving dependencies

**Comment by asottile** [comment]
CC @chriskuehl

**Comment by solarkennedy** [comment]
Thanks!

**Pull request merged**